content/en/docs/components/pipelines/sdk/output-viewer.md

```diff
@@ -505,7 +667,17 @@ def tensorboard_vis(mlpipeline_ui_metadata_path: kfp.components.OutputPath()):
 <a id="example-source"></a>
 ## Source of examples on this page
 
-The above examples come from the *tax tip prediction* sample that is
+### v2 example
+
+The metric visualization in V2 or V2 compatible mode depends on SDK visualization APIs,
+refer to [metrics_visualization_v2.py](https://github.com/kubeflow/pipelines/blob/master/samples/test/metrics_visualization_v2.py)
+for a complete pipeline example. Follow instruction
+[Compile and run your pipeline](/docs/components/pipelines/sdk/v2/build-pipeline/#compile-and-run-your-pipeline)
+to compile in V2 compatible mode.
```

Comment:
suggest moving v2 example section next to v2 introduction.

Reply:
It makes total sense, I moved the v2 example to higher location. I have already included a link to this v2 example in v2 visualization introduction too.